atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweepThread.java

```diff
@@ -176,11 +177,13 @@ private void updateMetrics(SweepOutcome outcome) {
     }
 
     private void sleepUntilNextRun(SweepOutcome outcome) throws InterruptedException {
-        long sleepDurationMillis = getBackoffTimeWhenSweepHasNotRun();
+        Duration sleepDuration = getBackoffTimeWhenSweepHasNotRun();
```

Comment:
nit: might be cleaner as a switch statement rather than splitting out the functions